src/olympia/devhub/templates/devhub/addons/includes/static_theme_preview_svg.xml

```diff
@@ -1,35 +1,64 @@
 {% autoescape true %}
-<svg id="SvgjsSvg1006" width="900" height="200" xmlns="http://www.w3.org/2000/svg"
+<svg id="Svg1006" width="680" height="100" xmlns="http://www.w3.org/2000/svg"
      version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink"
-     xmlns:svgjs="http://svgjs.com/svgjs" font-size="18px" font-family="Helvetica, Arial, sans-serif">
-    <defs id="SvgjsDefs1007"></defs>
-    <rect id="SvgjsRect1008" width="900" height="200"
-          class="accentcolor" fill="{{ accentcolor|d('#ccc') }}" data-fill="#e3e4e6"></rect>
-    <image id="svg-header-img" width="900" height="200" preserveAspectRatio="xMaxYMin slice"
+     xmlns:svgjs="http://svgjs.com/svgjs" font-size="16px" font-family="Helvetica, Arial, sans-serif">
+    <defs id="SvgDefs1007"></defs>
+    <rect id="SvgRect1008" width="680" height="100"
+          class="accentcolor" fill="{{ accentcolor|d('rgba(229,230,232,1)') }}" data-fill="rgba(229,230,232,1)"></rect>
+    <image id="svg-header-img" width="680" height="200" preserveAspectRatio="xMaxYMin slice"
            xlink:href="{{ header_src|d }}"></image>
-    <text id="SvgjsText1010" x="5" y="34" class="textcolor" fill="{{ textcolor|d }}">
-        <tspan id="SvgjsTspan1011" dy="1.8199999999999998" x="5">
-            {{ _('File    Edit   View    History   Bookmarks   Tools   Help') }}
-        </tspan>
+    <text id="SvgText1012" x="200" y="25" class="textcolor" fill="{{ textcolor|d }}">
+        <tspan id="SvgTspan1013" dy="2">֍ .... ....</tspan>
```

Comment:
`֍` ? This character is not displaying correctly on my machine.

Reply:
Alternate suggestion: pick one of 🌍🌎🌏 for each tab, or use 🌐 for all ?

Reply:
hmm, might be a windows-only glyph. I can use `🌐` instead